### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by yersel500**
I develop the desktop version of my portfolio project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -74,8 +74,8 @@ <h2>LET'S CONNECT</h2>
 
     <section class="project-wrap">
       <div class="project project1">
-        <img src="./images/portfolio-1.png" alt="My first project" />
-        <div class="description">
+        <img class="image image1" src="./images/portfolio-desktop/portfolio-1.png" alt="My first project" />
+        <div class="description description1">
```
Comment (rotimiazeez) [inline index.html:78]: - [x] Kindly make sure the work section buttons design match that on Figma as required😊
Your Design | Figma Design
:---:|:---:
 |

File: index.html [hunk 2]
```diff
@@ -215,6 +215,8 @@ <h4>LET'S CONNECT</h4>
         <div class="button">
           <button class="button1" type="button">Get my resume</button>
         </div>
+      </div>
+      <div class="third the-languages">
```
Comment (rotimiazeez) [inline index.html]: - [x] You are missing some images and icons in the about me section, kindly include them🙄👍
Your Implementation | Figma Design
:---:|:---:
 |

Comment (rotimiazeez) [reply index.html]: - [x] As stated earlier Kindly make sure your buttons styling matches that of Figma😉.

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi Team👋, 
## Good job so far!
### Summary 👏
- No linter errors ✔
- Good commit message 👍
- Descriptive PR message😊
- Good Readme 🚀
#### There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShoiraTa — APPROVED** [review]
### **STATUS: APPROVED  ✔️** 
------
Hi team,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 



Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**